pkgs/dash_design/lib/styles/variables.scss

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// styles/variables.scss:
+///
+/// Mixins that can be included to load CSS custom properties (variables)
+/// into any desired scope.
+
+/// CSS custom properties (variables) to be used across all themes.
+@mixin shared {
+  // Shared typography style variables.
+
+  /// Used for body text and smaller UI elements, like labels.
+  --dash-default-font-family: 'Roboto', sans-serif;
+
+  /// Used for headlines, titles, and larger UI elements.
+  --dash-display-font-family: 'Roboto', sans-serif;
+
+  /// Used for monospace text, such as code blocks.
+  --dash-mono-font-family: 'Roboto Mono', 'Source Code Pro', 'Cascadia Mono', 'JetBrains Mono', monospace;
+
+  /// Used for icons, such as the magnifying glass for search.
+  --dash-symbol-font-family: 'Material Symbols', 'Material Symbols Outlined';
+}
+
+/// CSS custom properties (variables) to be used for a light theme.
+@mixin light-theme {
+  /// The default text color, used for body text.
+  --dash-on-background-text-color: #212121;
```

Comment:
I would have to see it on a staging site or screenshot to really have any opinion here.

Reply:
I'll update this PR to use `#4a4a4a` for the purposes of this PR getting an initial structure in place.

I'll need to look back at why we moved away from it on the Dart site. It might be a bit too soft and low contrast (≈9 vs ≈16) for long text. Perhaps there's something in the middle that feels right.